examples/Demo/FluentUI.Demo.Client/Documentation/Components/Spacing/Spacing.md

```diff
@@ -144,3 +144,26 @@ These values can be customized or overridden in your own CSS files.
 --spacingHorizontalXXXL:  28px;
 --spacingHorizontalXXXXL: 32px;
 ```
+
+## Appendix - Boostrap
+
+If you use **Bootstrap** (or **MudBlazor**) in your project, we use the same class names
+(for consistency and ease of use). In this case, you choice to use **Bootstrap** or **FluentUI Blazor**
+classes for margins and paddings. You need to check the loading order of the **Styles** files in your main page:
```

Comment:
When using **Bootstrap** or **MudBlazor** (and possibly others) in your project, you will notice we use the same class names in some cases (this is done for consistency and ease of use). In this case, you make the choice to use **Bootstrap** or **FluentUI Blazor** classes for margins and paddings. You do this by setting the loading order of the **Styles** files in your main page: